doc/releases/changelog-0.30.0.md

```diff
@@ -149,53 +288,115 @@
   are real, providing a significant performance improvement.
   [(#3915)](https://github.com/PennyLaneAI/pennylane/pull/3915)
 
-* Three new decomposition algorithms have been added for n-controlled operations with a single-qubit target
-  and are selected automatically when they produce a better result, i.e., fewer CNOT gates.
-  They can be accessed via `ops.op_math.ctrl_decomp_bisect`.
-  [(#3851)](https://github.com/PennyLaneAI/pennylane/pull/3851)
+* The `coefficients` function and the `visualize` submodule of the `qml.fourier` module
+  now allow assigning different degrees for different parameters of the input function.
+  [(#3005)](https://github.com/PennyLaneAI/pennylane/pull/3005)
+
+  The arguments `degree` and `filter_threshold` to `qml.fourier.coefficients` previously were
+  expected to be integers, and now can be a sequences of integers with one integer per function
+  parameter (i.e. `len(degree)==n_inputs`), resulting in a returned array with shape
+  `(2*degrees[0]+1,..., 2*degrees[-1]+1)`.
+  The functions in `qml.fourier.visualize` accordingly accept such arrays of coefficients.
+
+<h3>Improvements 🛠</h3>
+
+<h4>Next-generation device API</h4>
+
+In this release and future releases, we will be making changes to our device API with the goal in mind to make 
+developing plugins much easier for developers and unlock new device capabilities. Users shouldn't yet feel any of 
+these changes when using PennyLane, but here is what has changed this release:
+
+* Several functions in `devices/qubit` have been added or improved:
+  + `sample_state`: returns a series of samples based on a given state vector and a number of shots. [(#3720)](https://github.com/PennyLaneAI/pennylane/pull/3720)
+  + `simulate`: supports measuring expectation values of large observables such as `qml.Hamiltonian`, `qml.SparseHamiltonian`, and `qml.Sum`. [(#3759)](https://github.com/PennyLaneAI/pennylane/pull/3759) 
+  + `apply_operation`: supports broadcasting. [(#3852)](https://github.com/PennyLaneAI/pennylane/pull/3852)
+  + `adjoint_jacobian`: supports adjoint differentiation in the new qubit state-vector device. [(#3790)](https://github.com/PennyLaneAI/pennylane/pull/3790)
```

Comment:
I think the nested bullet points are failing to render :(

Reply:
nooooooo

Reply:
```suggestion
  - `sample_state`: returns a series of samples based on a given state vector and a number of shots. [(#3720)](https://github.com/PennyLaneAI/pennylane/pull/3720)
  - `simulate`: supports measuring expectation values of large observables such as `qml.Hamiltonian`, `qml.SparseHamiltonian`, and `qml.Sum`. [(#3759)](https://github.com/PennyLaneAI/pennylane/pull/3759) 
  - `apply_operation`: supports broadcasting. [(#3852)](https://github.com/PennyLaneAI/pennylane/pull/3852)
  - `adjoint_jacobian`: supports adjoint differentiation in the new qubit state-vector device. [(#3790)](https://github.com/PennyLaneAI/pennylane/pull/3790)
```

In v0.27 release notes, `-` worked for nesting bullet points!

Reply:
Fixed: 676b79d2e19c078d0d8f53bdc4d6ee39e68d9354